web/router.ex

```diff
@@ -35,6 +35,11 @@ defmodule Healthlocker.Router do
     put "/account/update", AccountController, :update
     get "/account/consent", AccountController, :consent
     put "/account/update-consent", AccountController, :update_consent
+    get "/account/security", AccountController, :security
+    get "/account/edit-security", AccountController, :edit_security
+    put "/account/update-security", AccountController, :update_security
+    get "/account/edit-password", AccountController, :edit_password
```

Comment:
also what is the difference in `user story` for `edit-password` vs `update-password` ...?